emails/migrations/0038_profile_auto_block_spam.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.24 on 2022-01-11 21:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emails', '0037_reply_add_index_on_created_at'),
```

Comment:
I'll update this and the migration number + name after we merge https://github.com/mozilla/fx-private-relay/pull/1437.